include/wx/sizer.h

```diff
@@ -106,6 +106,11 @@ class WXDLLIMPEXP_CORE wxSizerFlags
 
     // default border size used by Border() below
     static int GetDefaultBorder()
+    {
+        return wxRound(GetDefaultBorderDouble());
+    }
+
+    static double GetDefaultBorderDouble()
```

Comment:
It's a nitpick but I wonder if we can use something better than "Double" in the new function name. Unfortunately I don't have any great ideas myself, something like `GetDefaultBorderFractionalPx()` is probably too long, but maybe still worth it?

Another idea would be to add `static int GetScaledDefaultBorder(int multiple)` that would avoid using `double` in the interface completely and use them only internally.

Reply:
I will rename it. It can also cause confusion with `DoubleBorder()` where double has a different meaning.